app/store/mods-state/mod-dependencies-state.ts

```diff
@@ -6,6 +6,9 @@ import { localModMap } from './mods-state';
 export const missingDependencyIdsState = selectorFamily({
   key: 'MissingDependencyIds',
   get: (mod: Mod) => ({ get }) => {
+    if (mod.isAlpha) {
+      return [];
+    }
```

Comment:
Can you explain this? Can alpha mods not have missing dependencies?

Reply:
Oops, this was a mistake. I forgot to remove it.